packages/office-ui-fabric-react/src/components/Callout/CalloutContent.tsx

```diff
@@ -314,20 +340,23 @@ export class CalloutContent extends BaseComponent<ICalloutProps, ICalloutState>
       if (this.props.directionalHintFixed && this._target) {
         let beakWidth = this.props.isBeakVisible ? this.props.beakWidth : 0;
         let gapSpace = this.props.gapSpace ? this.props.gapSpace : 0;
-        this._maxHeight = getMaxHeight(this._target, this.props.directionalHint!, beakWidth! + gapSpace, this._getBounds());
+        // Since the callout cannot measure it's border size it must be taken into account here. Otherwise it will
+        // overlap with the target.
+        const totalGap = gapSpace + beakWidth! + BORDER_WIDTH * 2;
```

Comment:
In my PR i actually renamed this to offsetFromTarget, i think gapSpace might be a bit ambiguous.

Reply:
We can deprecate that moving forward maybe but for now it needs to be kept as it's in the public api :/